ui/src/components/MarkdownMessage/MarkdownMessage.tsx

```diff
@@ -6,15 +6,37 @@ const MarkdownWrapper = styled.span`
     color: ${(props) => props.color};
 `;
 
-export interface MarkdownMessageProps {
+interface MarkdownMessageText {
```

Comment:
it looks like it can be inferred from the schemas in [entities.ts](https://github.com/splunk/addonfactory-ucc-generator/pull/1066/files#diff-3672f4a8b3196a7e9bf1f7a571c6f6472fc5fd511ebbc05e9687ca2d1f6d9e54)

Reply:
true, changed it to use infer